Tests/DistributedActorsTests/Cluster/MembershipGossipLogicSimulationTests.swift

```diff
@@ -19,10 +19,6 @@ import NIO
 import XCTest
 
 final class MembershipGossipLogicSimulationTests: ClusteredActorSystemsXCTestCase {
-    override func configureActorSystem(settings: inout ClusterSystemSettings) {
-        settings.enabled = false // not actually clustering, just need a few nodes
-    }
```

Comment:
Eh Oh Interesting... I guess maybe we do want to allow for this just of our internal tests then... :-\ 

Will think some more.